### ISSUE TITLE
Duplicating output with 2 Audio Interfaces to USB audio device freezes Rack

### ISSUE THREAD

**Issue by nbcallah** [VCVRack/Rack#354]
OS (if applicable): macOS Sierra 10.12.6
Version (or "dev" if compiling from source): 0.4.0
Other hardware: Sabrent USB audio device (may be necessary for replication) [a USB device with a stereo output and microphone input). May work with other devices.

Problem: Using 2 Audio Interface modules which both select the same USB audio device and sending a signal to both Audio Interface modules causes Rack to become unresponsive.

Replication:
* Open Rack.
* Create a VCO module
* Create an Audio Interface Module
* Select USB audio device from Audio Interface Module
* Route any output of VCO to Audio Interface Module
* Create a second Audio Interface Module
* Select USB audio device from Second Audio Interface Module
* Route output of VCO to Second Audio Interface Module in parallel
* Program should freeze

Since Rack automatically saves and re-starts sessions, it will cause the program to hang on startup after killing the application.

Temporary fix:
Delete the autosave.vcv in ~/Documents/Rack/

**Comment by bontric** [comment]
There is no need to create multiple Audio interfaces for the same device in the first place :wink: 

While Rack shouldn't crash either, the crash could be fixed with #82 .

**Comment by Euphorbium** [comment]
#132 duplicate

**Comment by AndrewBelt** [comment]
Yes, duplicate of #132
